cliutils/flagkit/flags.go

```diff
@@ -0,0 +1,201 @@
+package flagkit
```

Comment:
Since this is only used by distribution / lifecycle commands, I suggest placing it under a non common package.

Reply:
it is only used distribution, lifecycle since we have moved only those 2 categories for now, all command flags including artifactory or any other types if moved in future can be placed here hence the common package.

Removed unused flags from `jfrg-cli`